augur/application/schema/augur_full.sql

```diff
@@ -2777,6 +2777,35 @@ CREATE TABLE augur_data.working_commits (
 
 ALTER TABLE augur_data.working_commits OWNER TO augur;
 
```

Comment:
@ABrain7710 / @IsaacMilarky : Is this the right way to do this?

Reply:
The proper way to do this is with alembic which you did already. I would not do it this way.